src/components/training/UpcomingTrainings.js

```diff
@@ -82,7 +82,8 @@ const trainingByType = type => training => !type || training.type === type
 const trainingByTypes = types => training =>
   types && types.length ? types.find(type => type === training.type) : true
 
-const trainingByCity = city => training => !city || training.city === city
+const trainingByCity = city => training =>
+  !city || (training.city && training.city.toLowerCase()) === city.toLowerCase()
```

Comment:
is the city that's passed in now not a const? I guess this is to make it more Robust if that's the case, was just thinking that before we didn't need the `toLowerCase()`

Reply:
@ric9176  Now the city also comes from the .md file and it can't be a constant https://github.com/reactgraphqlacademy/reactgraphqlacademy/blob/locations/src/pages/locations/london/index.md. So in to make the code more robust in case we change the city in the md file I made the comparison no case sensitive